src/PatternkitDrupalTwigLib.php

```diff
@@ -86,8 +86,13 @@ public function getEditor($subtype = NULL,
       : $config;
     // @todo Move to own JS file & Drupal Settings config var.
     $markup = <<<HTML
-<div id="editor_holder"></div>
+<div id="magic-pixie-dust"></div>
 <script type="text/javascript">
+  let target = document.getElementById("magic-pixie-dust");
```

Comment:
Are we risking compatibility issues by using `let` here and not using a transpiler?

Reply:
yes.